src/screens/send/send.tsx

```diff
@@ -368,6 +368,7 @@ export class SendScreenComponent extends React.Component<
     }
 
     public async alertModalAddAddress() {
+        // TODO: check this, it's not opening all the time
```

Comment:
same problem as on Delete wallet?

Reply:
Same problem, yes. The modal does not open all the time.

Reply:
I had also tried with `await delay(0)`, but it didn't help.